### ISSUE TITLE
Typo or mis-insertion in api_device.php

### ISSUE THREAD

**Issue by khaytsus** [Cacti/cacti#4730]
https://github.com/Cacti/cacti/blob/8c2a6c3c7d0c0e64614376fc5521fc40004424b5/lib/api_device.php#L1214

This line should, presumably, be this: } elseif (!isset($raised[$poller_id]))     

Not

} elseif (!$isset(raised[$poller_id])) {

Looks like the isset got inserted in the wrong place, not sure.  Affects https://www.cacti.net/downloads/cacti-1.2.20.tar.gz as well

**Comment by netniV** [comment]
Yes, this is a duplicate but thanks for noticing so it can be closed quickly :)
